Storage/SQL/BrowserTable.swift

```diff
@@ -917,6 +917,14 @@ open class BrowserTable: Table {
             }
         }
 
+        if from < 25 && to >= 25 {
+            if !self.run(db, queries: [
+                "ALTER TABLE \(TableClients) ADD COLUMN version TEXT"
```

Comment:
This should be moved to `RemoteClientsTable`. Put it in an overriding implementation of `updateTable`. Leave the `BrowserTable` version at 24.